Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -1685,48 +1709,53 @@ private string GetImageCacheTag(BaseItem item, ImageType type)
             }
         }
 
+        /// <inheritdoc />
         public void ReportNowViewingItem(string sessionId, string itemId)
         {
             if (string.IsNullOrEmpty(itemId))
             {
                 throw new ArgumentNullException(nameof(itemId));
             }
 
-            //var item = _libraryManager.GetItemById(new Guid(itemId));
+            var item = _libraryManager.GetItemById(new Guid(itemId));
 
-            //var info = GetItemInfo(item, null, null);
+            var info = GetItemInfo(item, null);
 
-            //ReportNowViewingItem(sessionId, info);
+            ReportNowViewingItem(sessionId, info);
         }
 
+        /// <inheritdoc />
         public void ReportNowViewingItem(string sessionId, BaseItemDto item)
         {
-            //var session = GetSession(sessionId);
+            throw new NotImplementedException();
```

Comment:
eh?

Reply:
That GetSession method doesn’t exist anymore, and this method isn’t called anywhere afaik

Reply:
Why not remove it then?..

Reply:
We should try and understand it a bit more before removing it. This actually looks like a useful addition to the session manager if we add it back.

Reply:
@JustAMan I'm working on this endpoint right now so disregard any changes in this pull request.

Reply:
implemented in my pull request